.gitmodules

```diff
@@ -1,18 +1,18 @@
 [submodule "lib/p"]
 	path = lib/p
-	url = git@github.com:ambiata/p
+	url = git@github.com:markhibberd/p
```

Comment:
Submodule changes are probably less than ideal, but depend on 8.2/8.4 fixes from my forks. Going to try and snip these out on subsequent PRs anyway so may not matter, but if you want to pull in the master branches of those forks in some way, I could re-point to your forks if needed.

Reply:
This is fine by me - anything that keeps this stuff building is incredibly valuable. I'll give you push so we can keep it ticking over.

My eventual preference is to inline some of these ambiata libraries where necessary so nobody has to publish them. Started on this last year in https://github.com/thumphries/projector/compare/topic/deambiata but got burned out and stopped.